chef/data_bags/crowbar/bc-template-glance.schema

```diff
@@ -76,6 +76,7 @@
             "rbd": {
               "type": "map", "required": true, "mapping": {
                 "store_ceph_conf": { "type": "str", "required": true },
+                "store_admin_keyring": { "type": "str", "required": true },
```

Comment:
I'm not sure I understand -- with internal ceph, the values are known and can be hard-coded. That's my point: if not hard-coded, they could be changed by the user and that'd break stuff.